### PR TITLE
Us020 suspend respondent account

### PR DESCRIPTION
**Pull request by gemmai95**
Add implementation for suspending respondent account. Make BI case inactionable upon DEACTIVATED case event created and make a new B case if case group is not in COMPLETED or COMPLETEDBYPHONE.

### CODE REVIEW

**Comment by gemmai95** [comment]
[US020: INT Suspend respondents account (L) - Needed for R17](https://trello.com/c/sDXLtTMy/56-us020-int-suspend-respondents-account-l-needed-for-r17)

File: src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java [hunk 1]
```diff
@@ -241,11 +244,11 @@
    * To create a case event being given a parent case and json to describe the
    * case event to be created
    *
-   * @param caseId the parent case
+   * @param caseId                      the parent case
```
Comment (tej99) [inline src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java]: Did you mean to add these tabs? They've been added to a lot of the docs not sure if it makes it easier to read or messy :/

File: src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package uk.gov.ons.ctp.response.casesvc.endpoint;
 
+import antlr.collections.List;
```
Comment (tej99) [inline src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java]: You shouldn't need this, line 38 imports java.util.List.

File: src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java (hunk 1 above)
Comment (gemmai95) [reply src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java]: Think IntelliJ auto adds them

**Comment by JamesGardiner** [comment]
Closing for now - leaving branch in place for future PRs

**Pull request closed without merging**